### ISSUE TITLE
[a11y] Cannot configure welcome page image

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1310]
**Describe the bug**
The welcome page image URL is not displaying

**To Reproduce**
1. Go to `freshfish` welcome page
2. See default image
3. Go to remote user prefs, see config

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/164731022-8fef9dc0-fbb1-4043-835c-e74710615024.png)

**Desktop (please complete the following information):**
 - OS: Windows 10, Edge, macOS, Edge

**Comment by maxxxxxdlp** [comment]
The welcome page image configuration has now moved to the user preferences panel
